src/iOS.Core/Controllers/LockPasswordViewController.cs

```diff
@@ -28,6 +28,10 @@ public abstract class LockPasswordViewController : ExtendedUITableViewController
         private bool _pinLock;
         private bool _biometricLock;
         private bool _biometricIntegrityValid = true;
+        private int _invalidPinAttempts;
```

Comment:
`_invalidPinAttempts` should no longer exist

Reply:
Yea it shouldn't be there.